repl/src/main/scala/ammonite/repl/interp/Interpreter.scala

```diff
@@ -97,18 +99,27 @@ class Interpreter(prompt0: Ref[String],
                 .combinePrints(${printSnippet.mkString(", ")})
         """,
         printer,
+        fileName,
         extraImports
       )
 
   }
 
-  def processModule(code: String) = processScript(
+  def processModule(code: String, fileName: String = "Main.scala") = processScript(
     skipSheBangLine(code),
-    (code, imports) => withContextClassloader(eval.processScriptBlock(code, imports, printer))
+    (code, imports) => withContextClassloader(eval.processScriptBlock(code,
```

Comment:
Could we instead format this as

```scala
(code, imports) => 
  withContextClassloader(
    eval.processScriptBlock(code, imports, printer, filename)
  )
```

A bit of a nitpick but I think it looks a lot better